docs/Installation.md

```diff
@@ -101,38 +104,39 @@ For testing on a local machine, you can edit zonemaster.conf and change the "*:8
 to the host's IP or using localhost as ServerName if that is appropriate.
 
 
-* Reload apache
+#### Reload apache
 
 ```sh
 sudo systemctl enable apache2
 sudo systemctl reload apache2
 ```
 
-### 4. FreeBSD
+### 3. FreeBSD
 
-* For all commands below become root:
+For all commands below become root:
 
 ``su -l``
 
-* Install Apache (see [tutorial on Apache on FreeBSD]) and its dependencies:
+#### Install Apache (see [tutorial on Apache on FreeBSD]) and its dependencies
```

Comment:
It is better to keep the headline clean:
```
#### Install Apache and its dependencies
See [tutorial on Apache on FreeBSD].
```

Reply:
Fair enough.